ui/components/app/confirmation-warning-modal/confirmation-warning-modal.js

```diff
@@ -21,11 +21,11 @@ import {
   IconName,
   IconSize,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   Text,
 } from '../../component-library';
+import { ModalContent } from '../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../component-library/modal-header/deprecated';
 
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 2 34 38 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/627ff43b-b779-4997-bca5-002311d51bfc">

### After
<img width="1512" alt="Screenshot 2023-12-10 at 2 34 48 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/0a1459e1-78ff-40d0-b990-941f98661074">